sdk/key_vault/src/secret.rs

```diff
@@ -138,7 +138,7 @@ impl<'a, T: TokenCredential> KeyClient<'a, T> {
     ///
     /// Runtime::new().unwrap().block_on(example());
     /// ```
-    pub async fn get_secret(&mut self, secret_name: &'a str) -> Result<KeyVaultSecret, Error> {
+    pub async fn get_secret<'b>(&mut self, secret_name: &'b str) -> Result<KeyVaultSecret, Error> {
```

Comment:
This does not require manual lifetime ascription, instead we should do:
```suggestion
    pub async fn get_secret(&mut self, secret_name: &str) -> Result<KeyVaultSecret, Error> {
```

Reply:
looks good, I've applied this in all locations